app/src/main/res/values-en/strings.xml

```diff
@@ -459,11 +459,11 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="achievement_car_title">On the Road</string>
     <string name="achievement_car_solved_X">You solved %d quests that improve detail on the street infrastructure, putting the “street” in OpenStreetMap!</string>
     <string name="achievement_rare_title">Treasure Hunt</string>
-    <string name="achievement_rare_solved_X">You found and solved %d quests that are completed really rarely, congratulations!</string>
+    <string name="achievement_rare_solved_X">You found and solved %d quests that are rarely completed, congratulations!</string>
     <string name="achievement_citizen_title">Citizen</string>
-    <string name="achievement_citizen_solved_X">You solved %d quests useful for everyday life in town!</string>
+    <string name="achievement_citizen_solved_X">You solved %d quests that are useful for everyday life in town!</string>
     <string name="achievement_outdoors_title">Outdoors</string>
-    <string name="achievement_outdoors_solved_X">You solved %d quests useful for outdoor-people!</string>
+    <string name="achievement_outdoors_solved_X">You solved %d quests that are useful for outdoor people!</string>
```

Comment:
“Outdoor people” sounds a bit odd to me. “Outdoors people” maybe? Or “useful for outdoor adventures”?

Reply:
I believe "outdoor enthusiast" is the current polite way to say this?

Reply:
Would be fine too

Reply:
```suggestion
    <string name="achievement_outdoors_solved_X">You solved %d quests that are useful for outdoor enthusiasts!</string>
```